run_sim.py

```diff
@@ -6,27 +7,37 @@
 from time import sleep
 from StatsParser import StatsParser
 from StatsAnalysis import *
+import pexpect
 
 SIM_RUN_TIME = 30
 OUTPUT_DIR = "./out/"
 BUILD = "./Debug"
 SIM_LOGGING_TIME = 5
+KEY = "fill_in_pass"
```

Comment:
Could move this out to a separate file, and just read from that file. That file could then be left out of the repo, and we wouldn't have to worry about someone accidentally committing sensitive info.